src/main/kotlin/blackjack/domain/game/BlackjackParticipant.kt

```diff
@@ -0,0 +1,16 @@
+package blackjack.domain.game
+
+import blackjack.domain.betting.BettingMoney
+import blackjack.domain.participant.Dealer
+import blackjack.domain.participant.Player
+
+class BlackjackParticipant private constructor(
+    val dealer: Dealer,
+    val players: List<Player>
+) {
+    companion object {
+        fun of(dealer: Dealer, names: List<String>, money: (String) -> (BettingMoney)): BlackjackParticipant {
```

Comment:
```suggestion
        fun of(
            dealer: Dealer, 
            names: List<String>, 
            money: (String) -> (BettingMoney)
        ): BlackjackParticipant {
```
파라미터가 긴 경우에는 개행하는 것을 권장합니다. 😄